CHANGELOG.md

```diff
@@ -132,3 +132,4 @@ Read more about it in the [docs](https://hasura.io/docs/1.0/graphql/manual/auth/
 - server: check expression in update permissions (close #384) (rfc #3750) (#3804)
 - console: show pre-release update notifications with opt out option (#3888)
 - console: handle invalid keys in permission builder (close #3848) (#3863)
+- server: unlock events the locked events fetched by the HGE when the HGE is scaled down (close #3548)
```

Comment:
I'm not sure I understand what this means? How is the user affected by this?

Reply:
I would say something like:  "manage processing events when HGE instance is gracefully shutdown"

Reply:
@codingkarthik Also, this has been added to `v1.2.0-beta.1` changelog entry. Pls put this at the top `Next release`